src/export/oneApp/mocks/session.ts

```diff
@@ -2,13 +2,16 @@ import { Session } from "../../../types";
 import { mockProposedLDCPassportData } from "./passport";
 
 export const mockSession: Session = {
+  id: "session123",
   data: {
-    id: "abc123",
+    id: "flow123",
```

Comment:
Lots of our mock data was quite misleading! `lowcal_session.data.id` _is_ the flow id. Updated throughout!